projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-expiring-plans-notice.tsx

```diff
@@ -46,6 +46,16 @@ const useExpiringPlansNotice = ( redBubbleAlerts: RedBubbleAlerts ) => {
 			productsEffected,
 		} ) || {};
 
+	const onCloseClick = useCallback( () => {
+		const cookieKey = isExpiredAlert
+			? `${ productSlug }--plan-expired`
```

Comment:
```suggestion
			? `${ productSlug }--plan_expired`
```

I believe this is what is causing the issue